src/cf.data.pre

```diff
@@ -2851,7 +2851,39 @@ DOC_START
 	based on the username or source address of the user making
 	the request.
 
-	tcp_outgoing_address ipaddr [[!]aclname] ...
+	tcp_outgoing_address <address> [[!]aclname] ...
+
+	The required address parameter must be one of the following:
+
+		IP: A literal IP address.
+
+		domain: A domain name that Squid attempts to resolve once, at each
+			(re)configuration time, using getaddrinfo(3) or a similar blocking
+			service. DNS resolution errors are fatal.
+
+		match_client_tcp_dst: This keyword instructs Squid to postpone
```

Comment:
I recommend `use_` instead of `match_` prefix because there is already a lot of "matching" going on in this context. As far as this client destination IP address is concerned, we are "using" it. I do not insist on this change.

Reply:
I wouldn't mind making the change, and updating docs to reflect it. 
@yadij - do you have an opinion on the naming here by any chance?

Reply:
`client_tcp_dst` seems clear enough.